src/main/java/com/sendgrid/helpers/mail/objects/Personalization.java

```diff
@@ -144,6 +145,22 @@ public void setSendAt(long sendAt) {
     this.sendAt = sendAt;
   }
 
+  @JsonProperty("dynamic_template_data")
+  public Map<String,Object> getDynamicTemplateData() {
+    if(dynamicTemplateData == null)
+       return Collections.<String,Object>emptyMap();
```

Comment:
Thanks for the feedback @dgranahan! 
I use curly brackets in if statements per default on my projects, but I didn't use here because I saw others methods on this class following this standard. I'll update the PR with this change. :+1: